about/index.html

```diff
@@ -221,7 +221,7 @@ <h5 class="about-max">Presentations &amp; Talks</h5>
                 </div>
             </div>
             
-            <div class="col-md-3 col-sm-6 col-xs-12 pt" data-key="http://www.ustream.tv/recorded/86187859"> 
+            <div class="col-md-3 col-sm-6 col-xs-12 pt" data-key="https://www.ustream.tv/recorded/86187859"> 
```

Comment:
That's fine. If they redirect to http, let them do it; this way they have control over the redirect and can skip it in the future. As long as their https link is not broken (i.e. there is a redirect served), we should prefer that.